packages/projects/default-project/projectEventHooks.ts

```diff
@@ -127,9 +128,34 @@ const config = {
         )
     )
   },
-  // onUpdate: (app: Application) => {
-  //   return installAvatarsFromProject(app, avatarsFolder)
-  // },
+  onUpdate: async (app: Application) => {
+    await Promise.all(
```

Comment:
can we move this to a helper function in server-core that takes in a list of files and a project name, such that we can re-use it for other projects that provide avatars?